crates/anvil-polkadot/tests/it/state_injector.rs

```diff
@@ -190,8 +190,8 @@ async fn test_set_balance() {
 
     assert_eq!(
         node.get_balance(alith, None).await,
-        // 1000 dollars
-        U256::from_str_radix("100000000000000000000000", 10).unwrap()
+        // Default balance - existential deposit
```

Comment:
let's remove these comments, as the ED is added on top.

I'm a bit confused about why the balance is decreasing when in reality we bumped it